Introduce generated config.h

mkroening opened the pull request:
Resolves #33.

This removes the build time from being saved in the build for reproducibility, introduces `config.h` which is being generated from `config.h.in` by meson and includes a few opinionated rearrangements to `AboutPanel`:
Before:
![Screenshot from 2020-04-25 19-58-59](https://user-images.githubusercontent.com/28776973/80287051-7ab33900-872f-11ea-92ff-79e61e033f4a.png)
After:
![Screenshot from 2020-04-25 19-59-56](https://user-images.githubusercontent.com/28776973/80287058-80a91a00-872f-11ea-9207-054cfeab9b1d.png)

Are you okay with the rearrangements or would you rather have version and email as separate row?

Note: Its possible to use [wxHyperlinkCtrl](https://docs.wxwidgets.org/trunk/classwx_hyperlink_ctrl.html) to make hyperlinks (homepage and email) clickable, but that's not a drop-in since the spacing is different from wxStaticText, so that might be interesting in the future.

mkroening committed:
About: Remove build date for reproducible builds

mkroening committed:
config.h: Generate and centralize

yannikschaelte reviewed (approved):
Looks good! I suggest keeping the e-mail separately, and listing all names under "contributors", I think that makes it clearer.

mkroening committed:
About: Rearrange version

mkroening commented:
Okay, I have split the last commit, rebased on latest `develop`, removed the author field and added a contributors field:
![Screenshot from 2020-04-26 17-22-58](https://user-images.githubusercontent.com/28776973/80311905-9b889680-87e2-11ea-8e34-b0718f23f4f7.png)
Is this the way you meant it, @yannikschaelte?

yannikschaelte commented:
Ah sorry, contributors is correct now, but I meant to keep the field "Author" before the "E-Mail". Then I appear twice, but that should be ok. "Author" could be renamed "Main developer | Hauptentwickler", as you prefer.

mkroening committed:
About: Add contributors

mkroening commented:
Done:
![Screenshot from 2020-04-26 18-09-48](https://user-images.githubusercontent.com/28776973/80313068-27052600-87e9-11ea-9fa8-169f840a8c81.png)
